Calculator.c

```diff
@@ -0,0 +1,127 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <string.h>
+
+#define MAX 100
```

Comment:
Consider dynamic memory allocation. As this may not be sufficient for large expressions.

Reply:
Okay I will change